test/test_data.h

```diff
@@ -59,6 +59,20 @@ static const Profiles g_vaProfiles = {
     VAProfileVP9Profile1,
     VAProfileVP9Profile2,
     VAProfileVP9Profile3,
+    VAProfileHEVCMain12,
+    VAProfileHEVCMain422_10,
+    VAProfileHEVCMain422_12,
+    VAProfileHEVCMain444,
+    VAProfileHEVCMain444_10,
+    VAProfileHEVCMain444_12,
+    VAProfileHEVCSccMain,
+    VAProfileHEVCSccMain10,
+    VAProfileHEVCSccMain444,
+    VAProfileAV1Profile0,
+    VAProfileAV1Profile1,
+    VAProfileHEVCSccMain444_10,
+    VAProfileProtected,
```

Comment:
Greetings team,

Was browsing through the tests and noticed this PR. Here are some comments to hopefully push it forward.

Seems like this should be split in three patches and enhanced as follows:
 - the HEVC entries should also be added to `g_vaHEVCProfiles` just below
 - `VAProfileAV1Profile0` and `VAProfileAV1Profile1` should be added to `g_vaAV1Profiles`, which then ought to be used in the respective tests
 - `VAProfileProtected` - same comments as AV1 apply
 
 Hope that helps o/